src/udev/udev-builtin-net_id.c

```diff
@@ -774,7 +774,9 @@ static int names_mac(sd_device *dev, struct netnames *names) {
         if (r < 0)
                 return r;
 
-        i = strtoul(s, NULL, 0);
+        r = safe_atolu(s, &i);
```

Comment:
In the previous commit, `safe_atolu_full(attr, 10, &type);` is used for parsing `type` attribute.

Reply:
I tried to match the behavior of the original `strtoul` calls. `strtoul` is called with `base=0` here, hence I used `safe_atolu`, which does the same thing.

Reply:
Yeah, but we should use the same parser for the same attribute, for consistency.

Reply:
All right, changed.